packages/bloc/lib/src/bloc.dart

```diff
@@ -10,23 +9,24 @@ import '../bloc.dart';
 /// and transforms them into a `Stream` of `States` as output.
 /// {@endtemplate}
 abstract class Bloc<Event, State> extends Stream<State> implements Sink<Event> {
-  final PublishSubject<Event> _eventSubject = PublishSubject<Event>();
+  final _eventController = StreamController<Event>();
```

Comment:
Should be a broadcast Stream (PublishSubject is broadcast) -- this is why you had to change the test to check for a different error message, but that was correctly telling you something!

Reply:
Ok, agreed. Many listeners inside `transformEvents` looks strange but it is possible. I will change it to broadcast stream. Thanks